### ISSUE TITLE
torchrun:command not f

### ISSUE THREAD

**Issue by LZH101** [LanglandsLin/ActCLR#9]
![N](https://github.com/LanglandsLin/ActCLR/assets/77476978/0c891c9a-ce37-43fc-b61b-ac7a37968d67)

hi , torchrun:commad not found
Can you help me take a look at this issue? thankyou

**Comment by LanglandsLin** [comment]
maybe you can try this command
`CUDA_VISIBLE_DEVICES=4,5,6,7  python -m torch.distributed.launch --nproc_per_node 4 --master_port 12356 main.py pretrain_aimclr --config ./config/ntu60/pretext/pretext_aimclr_xview_joint.yaml`

**Comment by LanglandsLin** [comment]
If training with a single card, remember to modify the learning rate to 0.1 as the batchsize is reduced to a quarter.

**Comment by LZH101** [comment]
Hello, please give me some advice. If I use a single GPU for training, should I execute the following command?
CUDA_VISIBLE_DEVICES=0 python -m torch.distributed.launch --nproc_per_node=1 main.py pretrain_actclr --config ./config/ntu60/pretext/pretext_actclr_xview_joint.yaml



And, should my torch be 1.4.0? Thank you for your help.



1374149633
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "LanglandsLin/ActCLR"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年2月27日(星期二) 下午3:34
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [LanglandsLin/ActCLR] torchrun:command not f (Issue #9)





 
If training with a single card, remember to modify the learning rate to 0.1 as the batchsize is reduced to a quarter.

**Comment by LZH101** [comment]
> yes, and I think the version of torch should be all fine and I use 1.8 for training.

If I am using a single GPU, what command should I execute? thanks for your help.

**Comment by LanglandsLin** [comment]
I think this will work
`CUDA_VISIBLE_DEVICES=0 python -m torch.distributed.launch --nproc_per_node=1 main.py pretrain_actclr --config ./config/ntu60/pretext/pretext_actclr_xview_joint.yaml`

**Comment by LanglandsLin** [comment]
you can delete this code:
https://github.com/LanglandsLin/ActCLR/blob/23c62f5eff793bc5157da2ea6a22b6b4b7e5cd93/processor/processor.py#L175C2-L179C59

**Comment by 223d** [comment]
> you can delete this code: https://github.com/LanglandsLin/ActCLR/blob/23c62f5eff793bc5157da2ea6a22b6b4b7e5cd93/processor/processor.py#L175C2-L179C59

Thankyou very much
